lib/Test/MockFile.pm

```diff
@@ -237,9 +245,8 @@ sub _strict_mode_violation {
 sub import {
     my ( $class, @args ) = @_;
 
-    if ( grep { $_ =~ m/strict/i } @args ) {
-        add_file_access_hook( \&_strict_mode_violation );
-    }
+    grep { $_ eq 'nostrict' } @args
```

Comment:
`lc` ?

Reply:
I honestly missed the `/i` but I think it's better this way.